### PR TITLE
Add support for ubuntu

### PR DESCRIPTION
**Pull request by dale-c-anderson**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ galaxy_info:
   - name: EL
     versions:
     - 7
+  - name: Ubuntu
+    versions:
+    - xenial
```
Comment (dblenkus) [inline meta/main.yml]: Is this also tested on Xenial (16.04)?

File: README.rst [hunk 2]
```diff
@@ -12,8 +12,7 @@ modifications to ensure idempotency.
 Requirements
 ------------
 
-Currently, the role only supports `CentOS`_ and
-`Red Hat Enterprise Linux (RHEL)`_ EL7 distribution flavors.
+Currently, the role only supports `CentOS`_ , `Red Hat Enterprise Linux (RHEL)`_ EL7, `Ubuntu`_ LTS >= 18 distribution flavors.
```
Comment (dblenkus) [inline README.rst]: Please wrap the line at 79 characters to keep it readable in the editor.

File: README.rst [hunk 3]
```diff
@@ -44,6 +43,9 @@ verified. For more information, check
 .. _Amazon's instructions on verifying email addresses:
   http://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses.html
 
+.. _Ubuntu:
```
Comment (dblenkus) [inline README.rst]: It would be better to move it just under Red Hat to keep the same order as in the text.

**Review by dblenkus — COMMENTED** [review]
Thanks for your contribution! I only have a few minor comments before I can merge it.

Also, please squash both commits as they address the same thing.

File: meta/main.yml (hunk 1 above)
Comment (dale-c-anderson) [reply meta/main.yml]: Nope. Removed from meta.

File: README.rst (hunk 2 above)
Comment (dale-c-anderson) [reply README.rst]: Updated.

File: README.rst (hunk 3 above)
Comment (dale-c-anderson) [reply README.rst]: Fixed.

**Comment by dale-c-anderson** [comment]
@dblenkus thanks for the cleanup suggestions, those make sense. 

I also added that missing regexp terminator.

If you want me to squash from my end it means a new PR, which I can do if you want, but Github should give you the option to "squash + merge" which will accomplish the same thing.

image

**Comment by dblenkus** [comment]
Thanks for your contribution! I squashed it into 2 commits and merged it into master.

**Pull request closed without merging**